Version/Version.swift

```diff
@@ -269,7 +277,7 @@ extension NSProcessInfo {
     extension UIDevice {
         /// The current version of the operating system.
         public func systemVersion() -> Version? {
-            return Version(self.systemVersion())
+            return Version(self.systemVersion()!) //Is it recursive?
```

Comment:
No, it shouldn't be. `UIDevice` defines a property `systemVersion`. I think this needs to be updated so it uses the `String` property instead.